clusterloader2/cmd/clusterloader.go

```diff
@@ -39,6 +40,8 @@ var (
 func initClusterFlags() {
 	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file")
 	pflag.IntVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", 0, "number of nodes")
+	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.Provider, "provider", "", "Cluster provider")
+	// TODO(krzysied): Add support for HA cluster with more than one master.
```

Comment:
This TODO should be moved somewhere else.
I thing that probably to the ClusterConfig definition.

Reply:
TODO moved to the ClusterConfig definition.